core/src/dird/get_database_connection.cc

```diff
@@ -32,15 +32,15 @@ namespace directordaemon {
 
 BareosDb* GetDatabaseConnection(JobControlRecord* jcr)
 {
+  bool try_reconnect = false;
   return DbSqlGetPooledConnection(
       jcr, jcr->impl->res.catalog->db_driver, jcr->impl->res.catalog->db_name,
       jcr->impl->res.catalog->db_user,
       jcr->impl->res.catalog->db_password.value,
       jcr->impl->res.catalog->db_address, jcr->impl->res.catalog->db_port,
       jcr->impl->res.catalog->db_socket,
       jcr->impl->res.catalog->mult_db_connections,
-      jcr->impl->res.catalog->disable_batch_insert,
-      jcr->impl->res.catalog->try_reconnect,
+      jcr->impl->res.catalog->disable_batch_insert, try_reconnect,
```

Comment:
you can also save the variable completely and write
```suggestion
      jcr->impl->res.catalog->disable_batch_insert, 
      false /* try_reconnect */,
```